Automattic/jetpack
# Jetpack AI: iterate over top-level actions of the block toolbar

On this PR:

* Remove the top-level actions from the toolbar
* Move the `Write with AI...` to the secondary toolbar section

Fixes https://github.com/Automattic/jetpack/issues/31040

## Proposed changes:

* Jetpack AI: iterate over top level actions of the block toolbar

### Other information:


## Jetpack product discussion



## Does this pull request change what data or activity we track or use?



## Testing instructions:






* Go to the block editor
* Create/edit an AI Assistant block instance
* Confirm the top-level actions aren't there
* Confirm the `Write with AI` button is in the secondary section

Before | After
------|------
<img width="681" alt="image" src="https://github.com/Automattic/jetpack/assets/77539/e2ead655-a725-491b-8b6c-9a2d46879676"> | <img width="679" alt="image" src="https://github.com/Automattic/jetpack/assets/77539/0e501269-b51c-46d9-8829-bf520eb184dc">
